ostd/src/lib.rs

```diff
@@ -79,9 +79,6 @@ pub fn init() {
     let meta_vec = mm::init_page_meta();
     mm::page::allocator::init();
     mm::kspace::init_kernel_page_table(meta_vec);
-    // SAFETY: no CPU local objects have been accessed by this far. And
-    // we are on the BSP.
-    unsafe { cpu::cpu_local::init_on_bsp() };
```

Comment:
So where is it now?

Reply:
It is mistakenly added while rebasing, which is originated from former implementation and deprecated in the latest.